Projects/Claims/Sources/Views/SubmitClaim/SummaryInfo/SubmitClaimSummaryScreen.swift

```diff
@@ -10,114 +11,137 @@ public struct SubmitClaimSummaryScreen: View {
     public var body: some View {
         LoadingViewWithContent(.postSummary) {
             hForm {
-                VStack(alignment: .center) {
-
-                    displayTitleField()
-                    displayDateAndLocationOfOccurrenceField()
-                    displayModelField()
-                    displayDateOfPurchase()
-                    displayDamageField()
+                hSection {
+                    VStack(spacing: 8) {
+                        matter
+                        damageType
+                        damageDate
+                        place
+                        model
+                        dateOfPurchase
+                        purchasePrice
+                    }
+                }
+                .withHeader {
+                    HStack {
+                        L10n.changeAddressDetails.hTextNew(.body).foregroundColor(hLabelColorNew.primary)
+                        Spacer()
+                        Image(uiImage: hCoreUIAssets.infoSmall.image)
+                            .foregroundColor(hLabelColorNew.secondary)
+                    }
                 }
+                .sectionContainerStyle(.transparent)
             }
+            .hUseNewStyle
             .hFormAttachToBottom {
-                hButton.LargeButtonFilled {
-                    store.send(.summaryRequest)
-                } content: {
-                    hText(L10n.generalContinueButton)
+                VStack(spacing: 8) {
+                    NoticeComponent(text: L10n.claimsComplementClaim)
+                    hButton.LargeButtonFilled {
+                        store.send(.summaryRequest)
+                    } content: {
+                        hText(L10n.generalContinueButton)
+                    }
+                    .padding([.leading, .trailing], 16)
                 }
-                .padding([.leading, .trailing], 16)
             }
-
         }
+        .hUseNewStyle
     }
 
-    @ViewBuilder func displayTitleField() -> some View {
+    @ViewBuilder
+    private var matter: some View {
         PresentableStoreLens(
             SubmitClaimStore.self,
             getter: { state in
                 state.summaryStep
             }
         ) { summaryStep in
-            hText(summaryStep?.title ?? "", style: .title3)
-                .padding(.top, UIScreen.main.bounds.size.height / 5)
-                .foregroundColor(hLabelColor.secondary)
+            createRow(with: L10n.claimsCase, and: summaryStep?.title ?? "")
```

Comment:
I think this should be "Broken phone or whatever was chosen from claimsTriaging

Reply:
@sladan-hedvig I don't think we are getting that attribute from backend

Reply:
We can save it when we are starting the flow and show it using that.

Reply:
title is now "broken item"